bin/compile

```diff
@@ -13,6 +13,11 @@ mkdir -p $INSTALL_DIR
 
 echo "Unpacking XPDF binaries to $INSTALL_DIR..."
 tar -zxf xpdf.tar.gz -C $INSTALL_DIR
+# Historically, we only had version 3.03, which went into the main $INSTALL_DIR, ss
+# we'll leave it for backward-compatibility and sentimentality, but copy every binary.
+mkdir $INSTALL_DIR/3.03
+cp $INSTALL_DIR/pdf* $INSTALL_DIR/3.03
```

Comment:
I tried using symlinks for this at first, but CircleCI wasn't loving it. I tried wholesale copying just to debug, and since it worked, I didn't think saving the 25MB or so of binaries on the dyno disk space to copy was worth debugging to get symlinks working, but I will go back to it if you think otherwise.

Reply:
Why do we still need version 3.03 here? If you really need to preserve it (which I'm not convinced we need to), it would be better to change the tar command on line 15

Reply:
pdftotext versions are not backward compatible. Even when I tried v.3.04 years ago, it produced different line extraction results that broke various PDF parses. Are you saying that you're okay with this as is? Because your folder structure looks exactly as I intended. See below.

Note that The new version is NOT 4.0.4. [It's 4.04](https://www.xpdfreader.com/download.html) (I dunno why they zero-prefixed the minor version, but they did. I am too grateful to the maintainers for the headaches they've saved us by this software to complain.)

I 100% agree that once this is launched, we should focus on changing over all of the our app configs to use the explicit versioned folder binaries so that we can remove the unversioned set.

Is there any change you want me to make at this time, though?

<img width="418" alt="Screen Shot 2023-03-22 at 4 55 20 PM" src="https://user-images.githubusercontent.com/1094182/227035945-9f1cae48-9594-4cf7-a402-f09596fe5335.png">

Reply:
Yeah:

1. Update the -C flag on the tar commands to put these in specifically versioned folders
2. Change the tar in the root that's 3.03 version to include 3.03 in the name
3. Copy the 3.03 version to the root install_dir (without the version prefix)

Currently I think 4.04 will be installed into the root INSTALL_DIR folder, which will break backwards compatibility

Reply:
The 4.04 archive unpacks to a `4.04` folder already, so I can see why you assumed it would dump its binaries into the root. The 3.03 one is just the binaries and does do that - I will make both archives be just the binaries so that the tar commands are more consistent. But I don't understand what you mean by "include 3.03 in the name" - in the name of the binaries, or the folders?

Do you mean you want it to look like this?
<img width="853" alt="Screen Shot 2023-03-22 at 6 53 02 PM" src="https://user-images.githubusercontent.com/1094182/227056093-0fcdc4bd-542f-42c5-8843-28161b792aea.png">

Reply:
Oh I see, so you packaged up the `xpdf-4.04` with a 4.04 folder already so that when you extract it it goes to the 4.04 folder? That's not so obvious so either:
1) clarify in comments / readme for next time we need to tar up a version
2) Only tar up the binaries and change the untar command to go into a specific folder (so it's clear in code)

But re. your question - the tar included in this repo currently is called `xpdf.tar.gz` -> let's update that to `xpdf-3.03.tar.gz` for consistency

Reply:
And your first screenshot is what I'd expect, assuming that all of the binaries at the root level are the 3.03 version

Reply:
The goal being, once this is updated, the only change to the buildpack is that there are 2 additional folders with the specific versions